frappe/public/js/legacy/clientscriptAPI.js

```diff
@@ -66,8 +66,19 @@ refresh_field = function(n, docname, table_field) {
 	if(typeof n==typeof [])
 		refresh_many(n, docname, table_field);
 
-	if(table_field && cur_frm.fields_dict[table_field].grid.grid_rows_by_docname) { // for table
-		cur_frm.fields_dict[table_field].grid.grid_rows_by_docname[docname].refresh_field(n);
+	if (n && typeof n==='string' && table_field){
```

Comment:
Hey Max, can you explain what you are trying to do in this section and why is this code necessary?